src/app/BecomeACaptainPage/components/InviteMemberModal/components/ContactContent/ContactContent.jsx

```diff
@@ -0,0 +1,167 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import validator from 'validator';
+import styled from 'styled-components';
+import Button from '../../../../../../components/Button';
+import ErrorMessage from '../../../../../../components/ErrorMessage';
+import useToggler from '../../../../../../hooks/useToggler/useToggler';
+import ContactBar from './components/ContactBar';
+import Anchor from '../../../../../../components/Anchor/Anchor';
+import Message from '../Message';
+
+const StyledContactContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 100%;
+`;
+
+const Notice = styled.div`
+  font-family: ZCOOL KuaiLe;
+  font-size: 18px;
+  margin: 0 auto;
+`;
+
+const Form = styled.form`
+  width: 100%;
+  margin-top: 60px;
+`;
+
+const Label = styled.label`
+  font-size: 20px;
+  letter-spacing: 2px;
+  align-self: flex-start;
+`;
+
+const TextAreaContainer = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  margin-top: 8px;
+`;
+
+const Textarea = styled.textarea`
+  background-color: #FCEAC6;
+  border-radius: 8px; 
+  padding: 4px 16px;
+  width: 420px;
+  height: 70px;
+  resize: none;
+  outline: none;
+  border: none;
+  font-size: 18px;
+  line-height: 30px;
+  letter-spacing: 1px;
+`;
+
+const ConfirmButtonWrapper = styled.div`
+  margin: auto 0;
+`;
+
+const BlockTitle = styled.div`
+  margin: 32px 0 12px 0;
+  padding-bottom: 12px;
+  width: 100%;
+  border-bottom: 1px solid #202020;
+  font-size: 20px;
+  letter-spacing: 2px;
+`;
+
+export default function ContactContent({
+  contacts, addContact, deleteContact,
+}) {
+  const [input, setInput] = useState();
+
+  const [touched, toggleTouched] = useToggler();
+
+  const [failed, toggleFailed] = useToggler();
+
+  const updateInput = (event) => {
+    event.preventDefault();
+
+    setInput(event.target.value);
+  };
+
+  const resetInput = useCallback(() => setInput(''), []);
+
+  const resetTouched = () => {
```

Comment:
里面的逻辑应该是 initial 逻辑。不需要太多 if 
```suggestion
  const intialForm = () => {
```